components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1103 @@
+
+The layer thickness of layer $k$, used in MPAS-Ocean, is
+
+$$
+h_k = \int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} dz.
+$$ (def-thickness)
+
+In Omega we will use the pseudo-thickness,
+
+$$
+{\tilde h}_k(x,y,t)
+&= \int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} d\tilde{z} \\
+&= \frac{1}{\rho_0} \int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho \, dz \\
+&= \frac{1}{\rho_0 g} \left( \hat{p}_{k+1}^{\text{top}} - \hat{p}_k^{\text{top}} \right)
+$$ (def-pseudo-thickness)
+
+which is the mass per unit area in the layer, normalized by $\rho_0$. This pseudo-thickness and layer-averaging will be used to express conservation laws in a mass-weighted coordinate system.  Pseudo-thickness, rather than geometric
+thickness will be the prognostic variable in Omega.
+
+The density-weighted average of any variable $\varphi({\bf x},t)$ in layer $k$ is
+
+$$
+{\overline \varphi}^{\tilde{z}}_k(x,y,t) =
+\frac{\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho \varphi dz}
+     {\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho dz}
+     =
+\frac{\frac{1}{\rho_0}\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho \varphi dz}
+     {\frac{1}{\rho_0}\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho dz}
+     =
+\frac{\int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} \varphi d{\tilde z}}
+     {\int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} d{\tilde z}}
+=
+\frac{1}{{\tilde h}_k}\int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} \varphi d{\tilde z}.
+$$ (def-layer-average)
+
+Rearranging, it is useful to note that
+
+$$
+\int_{\tilde{z}_{k+1}^{\text{top}}}^{\tilde{z}_k^{\text{top}}} \varphi d\tilde{z}
+ = {\tilde h}_k {\overline \varphi}^{\tilde{z}}_k(x,y,t).
+$$ (h-phi)
+
+This relation is frequently used in discretized fluxes and conservation equations to replace integrals with layer-mean quantities.
+
+## 8. Decomposition and Averaging
+
+### Decomposition
+
+In this document, two decompositions will be critical.  The first decomposition is
+
+$$
+\varphi = \overline{\varphi}^{\tilde{z}}_k + \delta \varphi
+$$ (vertical-decomposition)
+
+which is a density weighted vertical integral based upon [](#def-layer-average) and the deviation from this value.
+
+The second decomposition is
+
+$$
+\varphi = \left<\varphi\right> + \varphi^\prime
+$$ (reynolds-definition)
+
+which is the traditional Reynolds' average and deviation from this value.
+
+The fundamental ocean model equations are most often Reynolds' averaged to derive the sub gridscale stresses. Each variable is decomposed into an average and a fluctuating component [[](#reynolds-definition)]. The averaging operator is defined such that it can be passed through derivatives and integrals without corrections and an average of products with a single perturbation quantity is zero.
```

Comment:
Are we leaving the averaging operator general on purpose or can we specify the averaging variable early on for clarity? the implication here is spatial scale (e.g. subgrid scales) but the time-mean Reynolds averaging is as, if not more, common.